bin/generate_lib_csv.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+import argparse
+import os
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Generate the lib.csv for cellranger-arc.")
```

Comment:
Could you add a docstring explaining in 1-2 sentences what these config filese are and maybe link to a corresponding section on the 10x website?